lib/index.ts

```diff
@@ -9,10 +10,10 @@ var moduleNamespace = 'I18n';
 var destNamespacePath = path.join(destPath, moduleNamespace);
 const configJson = path.join(projectPath, 'i18n.json')
 
-function main () {
+export function main () {
     if (fs.existsSync(configJson)) {
         const rawJSON = fs.readFileSync(configJson);
-        const json = JSON.parse(rawJSON);
+        const json = JSON.parse(rawJSON.toString());
```

Comment:
this json could have an expected type like 
```ts
Partial<{ source: string, dest: string, namespace: string }>

Reply:
Thanks, this wasn't covered in my "Learn TS in 20 minutes" lessons :rofl: